site-manage/site-manage-util/util/src/java/org/sakaiproject/site/util/SiteConstants.java

```diff
@@ -33,6 +33,8 @@
 	
 	public static final String SORTED_BY_GROUP_TITLE = "group_title";
 	
+	public static final String SORTED_BY_DISPLAY_ID = "display_id";
```

Comment:
This constant should be more aptly named like "SORTED_BY_PARTICIPANT_DISPLAY_ID" as its sorting participants.